src/giphon/siphon.py

```diff
@@ -99,6 +97,10 @@ def siphon(
         "-v",
         help=("The level of verbosity."),
     ),
+    using_multiprocessing: Optional[bool] = Option(
```

Comment:
Instead of having a use_multiprocessing argument, we could have a `--parallel-workers` that defaults to zero, wdyt?